experiment/simulation/js/animation-basic-counter.js

```diff
@@ -0,0 +1,684 @@
+    
+    for (let index = 0; index < CLOCKDOT.length; index++) {
+        objectAppear(CLOCKDOT[index]);
+    }
+
+}
+function outputDisappear() {
+    //makes the output text disappear
+    
+    for (let index = 0; index < TEXTOUTPUT.length; index++) {
+        objectDisappear(TEXTOUTPUT[index]);
+    }
+
+}
+function outputVisible() {
+    //makes the output text appear
+    
+    for (let index = 0; index < TEXTOUTPUT.length; index++) {
+        objectAppear(TEXTOUTPUT[index]);
+    }
+
+}
+function jDisappear() {
+    //makes the J text disappear
+    
+    for (let index = 0; index < TEXTINPUT.length; index+=2) {
+        objectDisappear(TEXTINPUT[index]);
+    }
+    
+}
+function kDisappear() {
+    //makes the K text disappear
+    
+    for (let index = 1; index < TEXTINPUT.length; index+=2) {
+        objectDisappear(TEXTINPUT[index]);
+    }
+}
+
+
+function clockDisappear() {
+    //makes the clock text disappear
+    
+    for (let index = 0; index < TEXTCLOCK.length; index++) {
+        objectDisappear(TEXTCLOCK[index]);
+    }
+}
+
+function jVisible() {
+    //makes the J text appear
+    for (let index = 0; index < TEXTINPUT.length; index+=2) {
+        objectAppear(TEXTINPUT[index]);
+    }
+}
+function kVisible() {
+    //makes the K text appear
+    
+    for (let index = 1; index < TEXTINPUT.length; index+=2) {
+        objectAppear(TEXTINPUT[index]);
+    }
+}
+function clockVisible() {
+    //makes the clock text appear
+    for (let index = 0; index < TEXTCLOCK.length; index++) {
+        objectAppear(TEXTCLOCK[index]);
+    }
+}
+function fillColor(object,color){
+    gsap.set(object, {
+        fill: color
+    });
+}
+
+function clearObservation() {
+
+    OBSERV.innerHTML = "";
+}
+
+function allDisappear() {
+    jDisappear();
+    kDisappear();
+
+
+    jkDotDisappear();
+    clockDisappear();
+    clockDotDisappear();
+    outputDisappear();
+    for (let index = 0; index < OBJECTS.length; index++) {
+        fillColor(OBJECTS[index],"#008000");
+    }
+    
+
+
+}
+
+function outputHandlerSetter() {
+    //to set output dots
+    //this is called only once
+    
+    for (let index = 0; index < TEXTOUTPUT.length; index++) {
+        TEXTOUTPUT[index].textContent = 1;
+    }
+
+}
+
+function outputHandler() {
+    //changes the outputs
+    if (TEXTOUTPUT[1].textContent === "1" && TEXTOUTPUT[0].textContent === "1") {
+        TEXTOUTPUT[0].textContent = "0";
+        TEXTOUTPUT[1].textContent = "0";
+    }
+    else if (TEXTOUTPUT[1].textContent === "1" && TEXTOUTPUT[0].textContent ==="0") {
+        TEXTOUTPUT[0].textContent = "1";
+        TEXTOUTPUT[1].textContent = "0";
+    }
+    else if (TEXTOUTPUT[1].textContent === "0" && TEXTOUTPUT[0].textContent === "1") {
+
+        
+        TEXTOUTPUT[1].textContent = "1";
+    }
+    else if (TEXTOUTPUT[1].textContent=== "0" && TEXTOUTPUT[0].textContent === "0") {
+
+        TEXTOUTPUT[1].textContent = "1";
+    }
+
+}
+function set(object) {

```

Comment:
Please use a better name than set. I do not understand which property is being set here .